### PR TITLE
UserStory #22 - Display Info about Specialist

### LINKED ISSUE #163: SP: 3 Create view for Specialist information
Part of user story #22

### LINKED ISSUE #164: SP: 3 Create serializer for Specialist GET method
Part of user story #22

### LINKED ISSUE #166: SP: 3 Add tests for Specialist
Part of user story #22

### PR DESCRIPTION
**Pull request by shv833**
#163 - done/approved
#164 - done/approved
#165 - closed(reason: needless)
#166 - done/approved

### CODE REVIEW

File: beauty/api/models.py [hunk 1]
```diff
@@ -373,13 +373,13 @@ class Review(models.Model):
         CustomUser,
         on_delete=models.CASCADE,
         null=True,
-        related_name=_("Customer")
+        related_name="customer_reviews"
     )
```
Comment (morento101) [inline beauty/api/models.py:398]: Why did you kill localization :sob:

File: beauty/api/serializers/serializers_customuser.py [hunk 2]
```diff
@@ -206,6 +206,8 @@ class CustomUserDetailSerializer(PasswordsValidation,
         read_only=True,
         url_user_id="customer_id",
     )
+    specialist_reviews = serializers.HyperlinkedRelatedField(
+        many=True, view_name="api:review-get", read_only=True)
```
Comment (morento101) [inline beauty/api/serializers/serializers_customuser.py]: Is that mean that now all users, including Owner and Client, will have a specialist_reviews field in views where this serializer is used?

File: beauty/api/models.py (hunk 1 above)
Comment (morento101) [reply beauty/api/models.py:398]: Oh, now I understand, everything is good here

File: beauty/api/urls.py [hunk 3]
```diff
@@ -5,9 +5,12 @@
 app_name = "api"
 
 urlpatterns = [
-    path('', CustomUserListCreateView.as_view(), name='user-list'),
-    path('<int:pk>/', CustomUserDetailRUDView.as_view(), name='user-detail'),
-    path('<int:user>/order/<int:id>/',
+    path('user/', CustomUserListCreateView.as_view(), name='user-list'),
```
Comment (Misha86) [inline beauty/api/urls.py]: should be "users/"

**Review by Misha86 — CHANGES_REQUESTED** [review]

File: beauty/api/urls.py [hunk 4]
```diff
@@ -4,13 +4,17 @@
 app_name = "api"
```
Comment (pazuzu-ua) [inline beauty/api/urls.py]: Please change the quotes to double

**Review by pazuzu-ua — CHANGES_REQUESTED** [review]
Please release all conflicts, ensure flake8 and all will be good!

**Review by pazuzu-ua — APPROVED** [review]

**Review by therobotisnotatoy — APPROVED** [review]

**Review by therobotisnotatoy — APPROVED** [review]

**Review by YaroslavBorysko — APPROVED** [review]

**Review by datsiuk-ira — APPROVED** [review]

**Review by Misha86 — APPROVED** [review]

**Review by morento101 — APPROVED** [review]

**Review by Serhii-Voloshyn — APPROVED** [review]

**Review by OleksiiDatsiuk — APPROVED** [review]
Looks awesome!

**Pull request merged**